frontend/src/pages/geography.tsx

```diff
@@ -0,0 +1,825 @@
+      <Head>
+        <title>Geography</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Layout backgroundEffect>
+        <div className="container">
+          <div className="flex max-h-[523px] min-h-[523px]" id="map"></div>
+          <div className="flex flex-wrap items-center justify-center">
+            <div className="mb-24 flex max-w-[1280px] flex-col flex-wrap gap-4 pt-20 sm:max-h-[6273px] md:max-h-[4573px] lg:max-h-[3473px] xl:max-h-[2473px]">
+              {Array.from(letterMap.keys()).map((letter) => (
+                <LetterCard
+                  letter={letter}
+                  words={letterMap.get(letter) || []}
+                />
+              ))}
+            </div>
+          </div>
+        </div>
+      </Layout>
+    </>
+  );
+}
+
+const LetterCard = ({ words, letter }: { letter: string; words: string[] }) => (
+  <div className="flex h-fit max-h-fit max-w-[200px] grid-rows-1 flex-col gap-5 rounded-lg bg-white p-6 shadow-md xl:w-[200px]">
+    <Badge
+      className="h-12 w-12 items-center justify-center px-3"
+      variant={"success"}
+    >
+      <span className="text-[30px] font-extrabold uppercase">{letter}</span>
+    </Badge>
+    <div className="flex flex-col gap-3">
+      {words.map((word) => (
+        <span className="break-words text-[#6B7280]">{word}</span>
+      ))}
+    </div>
+  </div>
+);
+
+const style: any = {
+  id: "43f36e14-e3f5-43c1-84c0-50a9c80dc5c7",
+  name: "MapLibre",
+  zoom: 0.8619833357855968,
+  pitch: 0,
+  center: [17.65431710431244, 41.954120326746775],
+  glyphs: "https://demotiles.maplibre.org/font/{fontstack}/{range}.pbf",
+  layers: [
+    {
+      id: "background",
+      type: "background",
+      paint: {
+        "background-color": "rgb(228 250 238)",
+      },
+      filter: ["all"],
+      layout: {
+        visibility: "visible",
+      },
+      maxzoom: 24,
+    },
+    {
+      id: "coastline",
+      type: "line",
+      paint: {
+        "line-blur": 0.5,
+        "line-color": "white",
+        "line-width": {
+          stops: [
+            [0, 3],
+            [6, 6],
+            [14, 9],
+            [22, 18],
+          ],
+        },
+      },
+      filter: ["all"],
+      layout: {
+        "line-cap": "round",
+        "line-join": "round",
+        visibility: "visible",
+      },
+      source: "maplibre",
+      maxzoom: 24,
+      minzoom: 0,
+      "source-layer": "countries",
+    },
+    {
+      id: "countries-fill",
+      type: "fill",
+      paint: {
+        "fill-color": "#D1D5DB",
+      },
+      filter: ["all"],
+      layout: {
+        visibility: "visible",
+      },
+      source: "maplibre",
+      maxzoom: 24,
+      "source-layer": "countries",
+    },
+    {
+      id: "countries-boundary",
+      type: "line",
+      paint: {
+        "line-color": "rgba(255, 255, 255, 1)",
+        "line-width": {
+          stops: [
```

Comment:
I think you can play around a little bit with these line-widths, i dont know if the other developers agree but on the mockup they seem smoother and i think thats in part cause of them being thicker
![image](https://github.com/user-attachments/assets/02c9c8b3-a4ef-49e6-9fc9-b4ed6bf9d931)
![image](https://github.com/user-attachments/assets/814f3375-ad52-427c-9034-03744b112a87)